windows/src/engine/tsysinfo/si_overview.xslt

```diff
@@ -128,8 +128,46 @@
             <xsl:value-of select="@x64"/>
           </td>
         </tr>
+        <tr>
+          <th>Native Machine</th>
+          <td>
+            <xsl:choose>
+              <xsl:when test="@NativeMachine = '014C'">Intel 386</xsl:when>
```

Comment:
Where do these table values come from?

Reply:
They are documented at https://learn.microsoft.com/en-us/windows/win32/sysinfo/image-file-machine-constants (that link is in the .pas, not in the .xslt though)